float32 pressure
float32 altitude
float32 temperature
uint64 error_count
uint32 device_id	# Sensor ID that must be unique for each baro sensor and must not change
